src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -519,7 +519,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			for (let i = 0; i < codeCells.length; i++) {
 				let cellStatus = await this.runCell(codeCells[i]);
 				if (!cellStatus) {
-					return Promise.reject(new Error(localize('cellRunFailed', 'running cell id {0} failed', codeCells[i].id)));
+					return Promise.reject(new Error(localize('cellRunFailed', 'Run Cells failed - See error in output of the currently select cell for more information.')));
```

Comment:
How is this displayed to the user? I tried it myself and I didn't see any sort of message pop up. And generally I'd ask why we're displaying this message anyways - the users can already see there's an error from the output of the cell so additional messages just seems redundant.

Regardless if this is useful I'd update the text, the current iteration reads a little oddly. Suggest : 

"An error occurred while running cells - see the currently selected cell for more information."

And if you aren't already I'd suggest scrolling the window to the cell (first cell if multiple) that has a failure so the user can see immediately when it happens.

Reply:
The issue is when running all cells, we got feedback that it wasn't clear that execution had failed (since it's only on a specific cell, you need to track progress). Hence UX/PM asked us to add a notification.

To repro, use Ctrl+Shift+F5 to run all cells and leave one in a failing state (e.g. run `select` on a Python3 kernel)